optashift/image.yaml

```diff
@@ -0,0 +1,169 @@
+    - name: "CUSTOM_INSTALL_DIRECTORIES"
+      example: "custom,shared"
+      description: "A list of comma-separated directories used for installation and configuration of artifacts for the image during the S2I process."
+    - name: "CONTAINER_HEAP_PERCENT"
+      example: "0.5"
+      description: "Deprecated.  See JAVA_MAX_MEM_RATIO."
+    - name: "INITIAL_HEAP_PERCENT"
+      example: "0.5"
+      description: "Deprecated.  See JAVA_INITIAL_MEM_RATIO."
+    - name: "PROBE_DISABLE_BOOT_ERRORS_CHECK"
+      example: "true"
+      description: "Disable the boot errors check in the probes."
+    - name: "ENABLE_ACCESS_LOG"
+      example: "true"
+      description: "Enable the Access Log."
+    - name: "ENABLE_JSON_LOGGING"
+      example: "true"
+      description: "Enable JSON-formatted logging"
+    - name: "OPTASHIFT_ROSTERING_PERSISTENCE_DIALECT"
+      example: "org.hibernate.dialect.MySQL5Dialect"
+      description: "server persistence dialect "
+    - name: "OPTASHIFT_ROSTERING_PERSISTENCE_DATASOURCE"
+      example: "java:jboss/datasources/dbe"
+      description: "server persistence datasource for persistence.xml "
+ports:
+    - value: 8080
+    - value: 8443
+modules:
+      repositories:
+          - git:
+              url: https://github.com/jboss-openshift/cct_module.git
+              ref: master
+          - git:
+              url: https://github.com/jboss-container-images/jboss-kie-modules.git
+              ref: master
+      install:
+          - name: dynamic-resources
+          - name: s2i-common
+          - name: java-alternatives
+          - name: os-eap-s2i
+          - name: os-java-jolokia
+          - name: jolokia
+          - name: os-eap7-openshift
+          - name: os-eap71-openshift
+          - name: os-eap7-modules
+          - name: os-eap71-modules
+          - name: os-eap7-ping
+          - name: os-java-run
+          - name: os-eap-launch
+          - name: os-eap7-launch
+          - name: os-eap-logging
+          - name: os-eap-probes
+          - name: jboss-maven
+          - name: os-eap-db-drivers
+          - name: os-eap-sso
+          - name: os-eap70-sso
+          - name: os-java-hawkular
+          - name: os-eap70-hawkular
+          - name: os-eap-hawkular
+          - name: os-eap-deployment-scanner
+          - name: os-eap-extensions
+          - name: openshift-layer
+          - name: openshift-passwd
+          - name: os-logging
+          - name: os.optashift
+packages:
+      repositories:
+          - jboss-os
+          - jboss-rhscl
+      install:
+          - rh-mongodb32-mongo-java-driver
+          - postgresql-jdbc
+          - mysql-connector-java
+          - rh-maven33
+          - hostname
+          - python-requests
+          - python-enum34
+          - PyYAML
+artifacts:
+    - path: javax.json-1.0.4.jar
+      md5: 569870f975deeeb6691fcb9bc02a9555
+    - path: jboss-logmanager-ext-1.0.0.Alpha2-redhat-1.jar
```

Comment:
Please use "jboss-logmanager-ext-1.0.0.Alpha5-redhat-1.jar" , for more info see RHDM-604